examples/complete/providers.tf

```diff
@@ -0,0 +1,16 @@
+provider "aws" {
+  region = "us-east-1"
```

Comment:
we usually provide `region` in a var for the example
(which was deleted in `main.tf` but should be in `variables.tf` and set in TF defaults